prow/spyglass/README.md

```diff
@@ -100,7 +100,7 @@ deck:
     - lens:
         name: junit
       required_files:
-      - ^artifacts/junit.*\.xml$
+      - ^artifacts/junit_[^/]*\.xml$
```

Comment:
This would exclude a simple `junit.xml` file, no? Not sure I like that.

I'm not clear what the old regex picks up that it should not?

Reply:
That would be in accordance with Gubernator's docs to exclude `artifacts/junit.xml`. The page explicitly has an underscore in the filename.

The old regex matches `artifacts/junit/blah.xml` but TestGrid (and presumably Gubernator but untested) does not.

As I wrote in the beginning, Gubernator's page can also be changed. I just believe they should be consistent.

Reply:
Gubernator is deprecated so I'm fine breaking with its convention. These days we treat the layout described by testgrid as authoritative. So yeah we could change gubernator's docs I guess? I don't know if the code needs changes